src/env.rs

```diff
@@ -791,6 +847,65 @@ mod tests {
         assert_eq!(k.stat().expect("stat").leaf_pages(), 1);
     }
 
+    #[test]
+    fn test_info() {
+        let root = Builder::new().prefix("test_info").tempdir().expect("tempdir");
+        fs::create_dir_all(root.path()).expect("dir created");
+        let k = Rkv::new(root.path()).expect("new succeeded");
+        let sk: SingleStore = k.open_single("sk", StoreOptions::create()).expect("opened");
+        let mut writer = k.write().expect("writer");
+
+        sk.put(&mut writer, "foo", &Value::Str("bar")).expect("wrote");
+        writer.commit().expect("commited");
+
+        let info = k.info().expect("info");
+
+        // The default size is 1MB.
+        assert_eq!(info.map_size(), 1024 * 1024);
+        // Should greater than 0 after the write txn.
+        assert!(info.last_pgno() > 0);
+        // A txn to open_single + a txn to write.
+        assert_eq!(info.last_txnid(), 2);
+        // The default max readers is 126.
+        assert_eq!(info.max_readers(), 126);
+        assert_eq!(info.num_readers(), 0);
```

Comment:
Is it possible to test that this number increments if we create a read transaction?